README.md

```diff
@@ -34,7 +34,8 @@
 - Built-in graphics library
 - Easy cross compilation
 - REPL
-- Built-in ORM
+- [Built-in ORM](https://github.com/vlang/v/blob/master/doc/docs.md#orm)
+- [Developing web applications](https://github.com/vlang/gitly)
```

Comment:
```suggestion
- [Developing web applications](https://github.com/vlang/v/blob/master/vlib/vweb/README.md)
```

Reply:
fixed it :)
https://github.com/vlang/v/pull/6055/commits/406ed1abf6d296a1650808e781af36cd2189759f